src/main/scala/xiangshan/frontend/IFU.scala

```diff
@@ -302,18 +300,28 @@ class NewIFU(implicit p: Parameters) extends XSModule
   .elsewhen(f0_fire && !f0_flush) {f1_valid  := true.B}
   .elsewhen(f1_fire)              {f1_valid  := false.B}
 
-  val f1_pc_adder_cut_point = (VAddrBits/2) - 1 // equal lower_result overflow bit
-  val f1_pc_high            = f1_ftq_req.startAddr(VAddrBits-1,f1_pc_adder_cut_point)
+  // val f1_pc_adder_cut_point = (VAddrBits/2) - 1 // equal lower_result overflow bit
+  // This is the point calculated for splicing PC,(VAddrBits/2) - 1 is too large,pc+32.U only need 6 bits
+  val pc_adder_cut_point = (VAddrBits/4) - 1 // equal lower_result overflow bit
```

Comment:
1. `pc_adder_cut_point` is a Scala constant, maybe use camel-case is better, e.g. `def PcCutPoint = (VAddrBits/4) - 1`

Reply:
Thank you very much for your suggestion, I will correct it according to your suggestion.